_topic_maps/_topic_map.yml

```diff
@@ -595,47 +595,50 @@ Topics:
 - Name: Preparing to update to OKD 4.13
   File: updating-cluster-prepare
   Distros: openshift-origin
-- Name: Preparing to perform an EUS-to-EUS update
-  File: preparing-eus-eus-upgrade
-  Distros: openshift-enterprise
-- Name: Preparing to update a cluster with manually maintained credentials
-  File: preparing-manual-creds-update
-- Name: Updating a cluster using the web console
-  File: updating-cluster-within-minor
-- Name: Updating a cluster using the CLI
-  File: updating-cluster-cli
-- Name: Migrating to a cluster with multi-architecture compute machines
-  File: migrating-to-multi-payload
-- Name: Performing update using canary rollout strategy
-  File: update-using-custom-machine-config-pools
-- Name: Updating a cluster that includes RHEL compute machines
-  File: updating-cluster-rhel-compute
-  Distros: openshift-enterprise
-- Name: Updating a cluster in a disconnected environment
-  Dir: updating-restricted-network-cluster
-  Topics:
-  - Name: About cluster updates in a disconnected environment
-    File: index
-  - Name: Mirroring the OpenShift Container Platform image repository
-    File: mirroring-image-repository
-  - Name: Updating a cluster in a disconnected environment using OSUS
-    File: restricted-network-update-osus
+- Name: Updating a cluster
```

Comment:
nit: Do we want to change either or both of the top-level `Updating clusters` and this `Updating a cluster` to  make the scope distinction more clear?  The former includes context, complicated pre-minor-update setup, and the actual updates, while the later is just about the act of updating.  Using `Cluster updates` and `Updating a cluster` would get at that with some present-participle hinting at where the action is.  But adding more words to be more explicit  might be helpful too (`Performing a cluster update`?  Something else?).  "Nah, we're fine with the current wording" is also ok with me ;)

![image](https://github.com/openshift/openshift-docs/assets/209920/8d01e510-cf37-4a16-88b9-f672ec18d90f)

Reply:
I think changing the top level name would require a level of change management that I don't wish to pursue in this reorg effort, but I can easily rename this new section. My first instinct would also be Performing a cluster update